src/screens/ToggleGroup.js

```diff
@@ -20,9 +20,9 @@ const ToggleGroupPage = () => (
       name="ToggleGroup"
       availableAt={[
         {
-          url: '',
+          url: 'https://storybook.grommet.io/?path=/story/controls-togglegroup-simple--simple',
           badge:
-            'https://storybook.grommet.io/?path=/story/controls-togglegroup-simple--simple',
+            'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
```

Comment:
Where does this badge appear?

Reply:
I think we used to have icons next to the storybook and codesandbox links, but looks like those are not showing up. I don't remember if we intentionally removed them or not

Reply:
If I had to guess I would say we intentionally removed them, but then I'm not sure why we kept the Github icon

Reply:
yeah I looked in `componentDoc` and saw `badge` was never being used

Reply:
removed!